components/script/dom/htmloptgroupelement.rs

```diff
@@ -75,14 +74,13 @@ impl<'a> VirtualMethods for JSRef<'a, HTMLOptGroupElement> {
                     child.set_enabled_state(false);
                 }
             },
-            _ => ()
+            _ => (),
```

Comment:
Ah, yes. This was unintentional. @Ms2ger, would you like me to make another PR to remove this stray comma?